sdk/ml/azure-ai-ml/azure/ai/ml/entities/_autogen_entities/models/_models.py

```diff
@@ -0,0 +1,264 @@
+# coding=utf-8
+# pylint: disable=too-many-lines
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) Python Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from typing import Any, Dict, Mapping, Optional, TYPE_CHECKING, overload
+
+from .. import _model_base
+from .._model_base import rest_field
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from .. import models as _models
+
+
+class AzureOpenAIDeployment(_model_base.Model):
```

Comment:
All of these models will need to be added to `entities.__init__.py` to be made public.

Reply:
only exposing necessary models for now